README.md

```diff
@@ -1 +1,150 @@
+      <br>TypeScript
+    </td>
+    <td align="center">
+      <img width="80px" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/tailwindcss/tailwindcss-plain.svg" alt="Tailwind CSS" />
+      <br>Tailwind CSS
+    </td>
+    <td align="center">
+      <img width="80px" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/vscode/vscode-original.svg" alt="vscode" />
+      <br>Vscode
+    </td>
+    <td align="center">
+      <img width="80px" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/git/git-original.svg" alt="git" />
+      <br>Git
+    </td>
+    <td align="center">
+      <img width="80px" src="https://cdn.worldvectorlogo.com/logos/jwt-3.svg" alt="jwt" />
+      <br>JWT
+    </td>
+    <td align="center">
+      <img width="80px" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/github/github-original.svg" alt="github" />
+      <br>Github
+    </td>
+  </tr>
+</table>
+
+
+## Project Features
+
+- Home Page: Displaying information about the project including project overview, team members, and project design files.
+- Contact-us route: A way for users to get in touch with the platform administrators by sending a message to the admin email.
+- User Authentication: Secure signup and login process with options for Google or sign-up/sign-in form.
+- User Profile: Users can manage their personal information, profile pictures, and provide a valid location in Turkey.
+- Dish Listings: Cooks can post dishes along with images, descriptions, categories, and allergy information.
+- Order Management: Users can browse dishes, place orders, and view their order history.
+- Cart: Users can add and remove items from their cart before finalizing orders.
+- Admin Dashboard: Cooks and Admin have extra access for managing dishes and orders.
+
+
+## Project Design
+
+### Entity-Relationship Diagram (ERD)
+
+We began the project design phase by creating an Entity-Relationship Diagram (ERD) to visualize the database structure. The ERD helps us understand the relationships between different entities and how data flows through the system.
+Here's a simplified representation of ERD design:
+<br />
+🔗 [Database design](https://www.figma.com/file/2LFORVZMP6cjBH9JvXeVML/DB-Schema-Design?type=design&mode=design&t=gqa5Oq9e3ej3AX0t-1) 
+
+
+### Architecture Design
+
+Our project's architecture was meticulously planned to ensure scalability, maintainability, and efficient performance. We designed a high-level overview of the routes and endpoints that we implemented to provide a seamless user experience. The architecture encompasses:
+
+- **User Authentication:** Implemented using Passport.js with Google strategy.
+- **Routes and Endpoints:** Defined clear routes and endpoints for user authentication, dish management, orders, and user profiles.
+- **Database Integration:** Utilized MongoDB to store user data, dishes, and order information.
+- **Admin Dashboard:** Created a separate route for Admin to manage dishes and orders.
```

Comment:
In here maybe we can add the users.
**Admin Dashboard:** Created a separate route for Admin to manage users, dishes and orders.

Reply:
✅